workflows/chipseq.nf

```diff
@@ -508,14 +510,18 @@ workflow CHIPSEQ {
                 .map { meta, saf -> [ meta.id, meta, saf ] }
                 .set { ch_ip_saf }
 
+            // ch_ip_saf.dump()
+
```

Comment:
```suggestion
```

Reply:
Fixed in [277bd11](https://github.com/nf-core/chipseq/pull/279/commits/277bd119f4747960d0059df30f7a3cea8fccb625)